docs/Overview.md

```diff
@@ -8,7 +8,7 @@ Draft.js is a framework for building rich text editors in React, powered by an i
 
 Draft.js makes it easy to build any type of rich text input, whether you're just looking to support a few inline text styles or building a complex text editor for composing long-form articles.
 
-Draft.js was introduced at [React.js Conf](http://conf.reactjs.com/) in February 2016.
+Draft.js was introduced at [React.js Conf](https://conf.reactjs.org/) in February 2016.
```

Comment:
Please change the URL to https://conf2016.reactjs.org/schedule.html#rich-text-editing-with-react

Reply:
@claudiopro Changed :)